mastracode/src/web/.env.example

```
@@ -108,8 +108,24 @@ MASTRACODE_TENANT_VECTOR_URL_TEMPLATE=
+# Turso auto-provisioning (optional, alternative to the URL templates above).
+# When both a platform token and org are set, each tenant's own Turso database
+# is created on first access via the Turso Platform API, a scoped token is
+# minted per resolution, and the stable db-name/hostname mapping is persisted
+# in the app Postgres (APP_DATABASE_URL) so all replicas converge on one DB.
+# Requires APP_DATABASE_URL. Takes priority over local files but not over an
+# explicit MASTRACODE_TENANT_DB_URL_TEMPLATE.
+# ---------------------------------------------------------------------------
+# optional — Turso Platform API token (https://app.turso.tech, platform scope)
+MASTRACODE_TURSO_PLATFORM_TOKEN=
+# optional — Turso organization slug/name that owns the provisioned databases
+MASTRACODE_TURSO_ORG=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MASTRACODE_TURSO_ORG key should go before the MASTRACODE_TURSO_PLATFORM_TOKEN key

(UnorderedKey)